boards/arc/nsim/support/nsim_hs6x.props

```diff
@@ -0,0 +1,31 @@
+	maxlastpc=0
+	trace_enabled=0
+	nsim_isa_family=arc64
+	nsim_isa_core=1
+	arcver=113
+	nsim_isa_atomic_option=1
+	nsim_isa_shift_option=3
+	nsim_isa_code_density_option=2
+	nsim_isa_div_rem_option=2
+	nsim_isa_swap_option=1
+	nsim_isa_bitscan_option=1
+	nsim_isa_enable_timer_0=1
+	nsim_isa_enable_timer_1=1
+	nsim_isa_has_interrupts=1
+	nsim_isa_dual_issue_option=1
+	nsim_isa_div64_option=1
+	nsim_isa_mpy64=1
+	nsim_isa_mpy_option=9
+	nsim_hostlink=1
+	gverbose=0
+	nsim_isa_addr_size=64
+	nsim_isa_pc_size=64
+	nsim_isa_vec64=1
+	dcache=16384,64,2,a
```

Comment:
Cache sizes seem to not match between nSIM & MDB

Reply:
Done, thanks!